superset/viz.py

```diff
@@ -143,18 +143,14 @@ def query_obj(self):
         # potential conflicts with column that would be named `from` or `to`
         since = (
             extra_filters.get('__from') or
-            form_data.get("since") or
-            config.get("SUPERSET_DEFAULT_SINCE", "1 year ago")
+            form_data.get("since")
         )
 
         from_dttm = utils.parse_human_datetime(since)
-        now = datetime.now()
```

Comment:
removing this created issues where some dashboards had expression like `180 days` that would get flipped to be in the past